packages/web/lib/browser.js

```diff
@@ -37,17 +82,15 @@ const getBrowserScope = (parent) => ({
         key,
         functions: {
           page:
-            "Defines a page definition which matches on the regex passed in as the first argument, and which executes the rule set passed in as the second argument on every matching page",
-          visit: "Open a new tab/page with the given url",
+            "Instantiates a page definition which matches on the url-pattern passed in as the first argument, and which executes the rule set passed in as the second argument on every matching page",
+          visit:
+            "Open a new tab/page with the given url and checks for matches on that URL. If there are matches the corresponding ruleSets will be run. If there is no match The new tab/page is opened in the browser scope and no actions will be taken",
```

Comment:
this is very much stylistic, but you said "rule set" in the previous explanation, and "ruleSets" here. 

Let's try to use "RuleSets" (PascalCase) as much as possible, just for aesthetics

Reply:
this doesn't have to block merging your PR. we can clean this up later too 🤷‍♀️